swagger.json

```diff
@@ -4430,7 +4430,31 @@
                     "format": "int64",
                     "description": "The amount of driving time in violation in this cycle (in ms).",
                     "example": 50400000
-                  }
+                  },
+                  "onDutyMs": {
```

Comment:
Nit: this looks like a unix timestamp until you read the description. Can we call this `onDutyDurationMs`? 

Also, it's not clear whether this is the current shift, cycle, etc, unless you read the description. Maybe `currentCycle { onDutyDurationMs }`?

Reply:
The naming convention here is consistent with the naming convention in `hos_daily_logs`

Reply:
But I see that there's a few different time periods represented in this endpoint. @dsharrison any opinion on naming?

Reply:
Good question. I don't have a strong preference, but think we should favor consistency within our API.